doc/classes/PropertyTweener.xml

```diff
@@ -17,8 +17,14 @@
 				When called, the final value will be used as a relative value instead.
 				[b]Example:[/b] Move the node by [code]100[/code] pixels to the right.
 				[codeblock]
+				[gdscript]
 				var tween = get_tree().create_tween()
 				tween.tween_property(self, "position", Vector2.RIGHT * 100, 1).as_relative()
+				[/gdscript]
+				[csharp]
+				Tween tween = GetTree().CreateTween();
+				Tween.TweenProperty(this, "position", Vector2.Right * 100.0f, 1.0f).AsRelative();
```

Comment:
```suggestion
				tween.TweenProperty(this, "position", Vector2.Right * 100.0f, 1.0f).AsRelative();
```
`tween` is the Tween you created, but `Tween` is the class itself.

Reply:
So if I click commit suggestion, it will add in the change that you have suggested?

Reply:
Yes, but don't do that just yet. 

Godot likes PRs to have only a single commit, to keep the Git history cleaner. There are two ways to do that:
- Amend your PR as you work on it, described [here](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#modifying-a-pull-request).
- Make fixup commits (including committing suggestions using the GitHub UI), then at the end "squash" all your commits together, described [here](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase).

You can use either, or both. Usually I find it easier to use amend, and I manually copy and paste suggestions from GitHub to my text editor. But you can also use the GitHub UI to "commit suggestion" and then squash at the end after all the changes are made. It's up to you.

Reply:
Ok, then should I wait till the other review that is pending finishes?

Reply:
Up to you. I personally would; usually I like to apply the obvious changes (in this case, everything but the callable one) as soon as possible. The C#-specific review probably won't happen immediately.

Either amending or committing more commits is fine here. In both cases reviewers can see the current state of the PR, with all the previous changes combined.

Reply:
Yeah, commiting from the UI is the simplest.

And yeah, for the codeblocks you'll need to make the rest of the changes manually.

Reply:
Ok.  I am wondering then if I should just do all manual and keep amending the PR.  Either way, its late where I am so I'll get back to this tomorrow.